Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/team_leader.yml

```diff
@@ -188,6 +188,7 @@
         recommended: true
       - id: RMCPouchFirstAidPills
         name: first-aid pouch (pills)
+        recommended: true
```

Comment:
This one isn't supposed to be recommended, it's injectors and gauze pouches only
```suggestion
```

Reply:
FTL just doesn't get it recommended huh? a bit weird